### DISCUSSION TITLE
change in "test your screen event" flow

### DISCUSSION THREAD

**Discussion by noobiessss** [Nain57/Smart-AutoClicker#514] [General]
would be nice to be able to test your screen event without having to include an action.

**Comment by shmaungms** [comment]
I have thought the same thing.  I'd like to test the detection as soon as I make it to see if I have to increase the slider above 4% without having to add actions first and then test the event.

(I'd also love to see the % detection number for each event if there are multiple but that suggestion should probably have it's own thread)

**Reply by noobiessss** [reply]
Totally agree. Knowing the % rate on multiple screen event would be helpful.
